### PR TITLE
Hdfsc 50 exception handler

### PR DESCRIPTION
**Pull request by danpece**

### CODE REVIEW

File: src/main/java/org/mule/extension/hdfs/internal/error/HdfsExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public class HdfsExceptionHandler extends ExceptionHandler {
+
+    private static final String ERROR_CODE = " ERROR CODE: ";
+
+    @Override
+    public Exception enrichException(Exception exception) {
+        try {
+            throw exception;
+        } catch (InvalidRequestDataException e) {
```
Comment (wandrei95) [inline src/main/java/org/mule/extension/hdfs/internal/error/HdfsExceptionHandler.java]: the new exception should be returned not thrown

Comment (alexandruatomei9) [reply src/main/java/org/mule/extension/hdfs/internal/error/HdfsExceptionHandler.java]: +1 to this

File: src/main/java/org/mule/extension/hdfs/internal/operation/HdfsOperations.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 import static org.mule.runtime.extension.api.annotation.param.MediaType.APPLICATION_OCTET_STREAM;
 
-public class HdfsOperations extends BaseOperations {
```
Comment (alexandruatomei9) [inline src/main/java/org/mule/extension/hdfs/internal/operation/HdfsOperations.java]: also remove the BaseOperations class

File: src/main/java/org/mule/extension/hdfs/internal/error/HdfsExceptionHandler.java (hunk 1 above)
Comment (danpece) [reply src/main/java/org/mule/extension/hdfs/internal/error/HdfsExceptionHandler.java]: Done

File: src/main/java/org/mule/extension/hdfs/internal/operation/HdfsOperations.java (hunk 2 above)
Comment (danpece) [reply src/main/java/org/mule/extension/hdfs/internal/operation/HdfsOperations.java]: Done

**Pull request merged**